spec/requests/apartments_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe "Apartments", type: :request do
+  describe "GET /index" do
+    it 'returns a list of apartments' do
+    
+      user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+      user.apartments.create(
+        street: string,
+        city: string,
+        state: string,
+        manager: string,
+        email: string, 
+        price: string, 
+        bedrooms: integer, 
+        bathrooms: integer, 
+        pets: string,
+        image: text
+      )
```

Comment:
Before moving on fix up this object so that it actually has some data values not the boilerplate.